docs/application/web/guides/applications/addon.md

```diff
@@ -0,0 +1,202 @@
+
+**Figure: Addon Mechanism**
+
+![Addon mechanism](./media/addon_mechanism.png)
+
+<a name="compare"></a>
+## Comparing to Chrome Extensions
+
+In the way to reach here, you might think that there’s no difference from Chrome Extensions.
+
+1. Applying to Multiple Applications
+
+   Chrome Extensions are applied to multiple pages (or tabs) in an application, Chrome. However, addons are applied to multiple web applications in a platform, Tizen at this moment.
+
+2. Wide Range of Customization
+
+   Addons can customize with related to each web application’s life cycle events such as Prelaunch, Resume, Suspend or Quit. Addons get the notification once they register its own callback for a lifecycle event. That’s possible because they are run on WRTjs, also known as Web Runtime JS, which manages and controls from birth to death of web applications.
+
+<a name="api"></a>
+## API
+
+Addon APIs are provided to support the way to handle the window in both browser process and renderer process. More APIs will be added in the future. The renderer process here is only available when the addon itself has html files and they are loaded.
+
+**Table: Supported APIs**
+
+| Module                                   | API                                      | Description                              | Type                                                                        |
+| ---------------------------------------- | ---------------------------------------- | ---------------------------------------- | ---------------------------------------- |
+| window | loadURL<br>show<br>showMessageBox | This provides core operation for window manipulation | Browser |
+| messaging | onMessage<br>sendMessage | This provides the way to communicate between browser process and renderer.<br>Renderer is not the one of web application but the one for the addon’s own page | Browser<br>Renderer |
+
+<a name="packaging"></a>
+## Packaging
+
+To generate a package for an addon, developers can use Tizen Studio.
+
+### Characteristics of the addon packages
+
+- Extension : WGT<br>
+- Icon after installation : Not Shown<br>
+- Tag in config file : `<tizen:addon>`
+
+Here’s an example of config.xml
+
+   ```
+   <?xml version="1.0" encoding="UTF-8"?>
+   <widget xmlns:tizen="http://tizen.org/ns/widgets" xmlns="http://www.w3.org/ns/widgets" id="http://yourdomain/AddonSampleAd" version="1.0.0" viewmodes="maximized">
+       <tizen:addon package="TFh8kl6ztm" required_version="5.5"/>
+       <name>AddonSampleAd</name>
+       <tizen:profile name="mobile"/>
+   </widget>
+   ```
+
+`<tizen:addon>` is a child element of `<widget>` tag. You can set traits of a addon such as package id, required_version. For required version, you should set it as 4.0 or higher. However, we recommend to use 5.5 as addons are supported from Tizen 5.5 officially.<br>
+A part from it, you can get to know that `<tizen:application>` has been removed. `<tizen:addon>` is the replacement of it in addon packaging.
```

Comment:
Could we add this information as a note?

> **Note**
>
> `<tizen:application>` is removed. It is replaced with `<tizen:addon>` in addon packaging.

Reply:
Done